aikau/src/main/resources/alfresco/services/InfiniteScrollService.js

```diff
@@ -112,10 +130,19 @@ define(["dojo/_base/declare",
        * @returns {boolean}
        */
       nearBottom: function alfresco_services_InfiniteScrollService__nearBottom() {
-         var currentScrollPos = domGeom.docScroll().y,
-             docHeight = this.getDocumentHeight(),
-             viewport = domGeom.getContentBox(document.body).h;
-         return 0 >= (docHeight - viewport - currentScrollPos - this.scrollTolerance);
+         if (this.scrollElement)
+         {
+            var scrollPos = $(this.scrollElement).scrollTop();
+            var height = $(this.domNode).height();
```

Comment:
Again, why JQuery? What's wrong with just offsetHeight and/or scrollHeight ... at least then it's explicit which one is being used. Which one is being used with this code?